godotproject/Area2D.gd

```diff
@@ -0,0 +1,7 @@
+extends Area2D
+
+# Called when the node enters the scene tree for the first time.
+func _ready():
+	pass # Replace with function body.
+
```

Comment:
delete this script file?

Reply:
cant seem to find where that is, but I agree it should be deleted

Reply:
*Liam Neeson voice* I will find you, and I will kill you